lib/node/NodeSourcePlugin.js

```diff
@@ -37,33 +38,39 @@ NodeSourcePlugin.prototype.apply = function(compiler) {
 			if(localOptions.process) {
 				var processType = localOptions.process;
 				parser.plugin("expression process", function() {
-					return ModuleParserHelpers.addParsedVariable(this, "process", "require(" + JSON.stringify(getPathToModule("process", processType)) + ")");
+					return ModuleParserHelpers.addParsedVariable(this, "process", "require(" +
+					    JSON.stringify(path.relative(this.state.module.context, getPathToModule("process", processType))) + ")");
```

Comment:
Could you extract your change into a function? And in this function change `\` to `/` for windows.